readme_release.md

```diff
@@ -19,20 +19,36 @@ Then follow the instructions given by the script, and also:
 
 ## Other releases
 
-For a major release, same as minor, but major:
+### For a major release, same as minor, but major:
 ```
 python ./script_release.py major <canaltp_distant_repo_name>
 ```
 
-For hotfix:
+### For hotfix:
 ```
 python ./script_release.py hotfix <canaltp_distant_repo_name>
 ```
+A new branch has been created <release_x.yy.z> and the changelog is opened.
 Then the process is less automated (but still, instructions are given):
-* you have to populate the changelog
-* and the TAG yourself
-* also cherry-pick the commits you want to release
-
+* Populate the changelog :
+	Add the hotfix PR name and link to github (as for the automated changelog in the release process)
+* Cherry-pick the commits you want to release:
+	```
+	git cherry-pick <commit_id> # Each commit_id of the hotfix PR
+	```
+* Merge the content of the new release branch with the hotfix commits to the 'release' branch:
+	```
+	git checkout release		
+	git merge --no-ff <release_x.yy.z>
+	```
+* Tag the new release:
```

Comment:
minor: Not sure I remember correctly, but I think there you also have to populate PR name and link here properly, with enough blank lines to have a clean message like here : https://github.com/CanalTP/navitia/releases/tag/v2.58.0

Reply:
Enough being how many? Looks like I missed this part and every other hotfix releases. For ex:
https://github.com/CanalTP/navitia/releases/tag/v2.52.1

Reply:
Looks like it's one fully blank line :
using `git show v2.57.0` (which looks OK)
```bash
Version 2.57.0

  * Kraken: Add ptref shortcut between physical_mode and jpps  <https://github.com/CanalTP/navitia/pull/2417>
```

`git show v2.58.1` (less OK)
```bash
v2.58.1: Ed: add rtree to speedup stop_point association to admins <https://github.com/CanalTP/navitia/pull/2466>
```